### PR TITLE
Update uasd page from scratch

### LINKED ISSUE #5070: [UA-I] redo /uasd page from scratch
lots of uncommented, changes, update the [entire doc](https://docs.google.com/document/d/1vQNdVmpsB3iQP7SnJHxT3A8rbD5jSAzTEoYUOepR-SM/edit#)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- rebuild /uasd from scratch

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/uasd](http://0.0.0.0:8001/uasd)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1vQNdVmpsB3iQP7SnJHxT3A8rbD5jSAzTEoYUOepR-SM/edit#)

## Issue / Card

Fixes #5070

### CODE REVIEW

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 1]
```diff
@@ -9,413 +9,380 @@ context:
 
 Capitalised terms shall have meanings as defined in the [Definitions section](#uasd-definitions).
 
-Each service includes access to Canonical’s knowledge base and the support described below within the scope and subject to the exceptions detailed in this document.
-
-This document defines Canonical’s service offerings. Canonical will only provide the services defined in this document which are covered by a customer contract.
-
 <h2 id="uasd-ua-infrastructure">Ubuntu Advantage for Infrastructure</h2>
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: Should be "Ubuntu Advantage for Infrastructure (UA-I)"

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 2]
```diff
@@ -9,413 +9,380 @@ context:
...
 
-|Benefit included|Essential|Standard|Advanced|
-|----------------|:---------:|:--------:|:--------:|
-|Self-service customer care portal and knowledge base|<img src="https://assets.ubuntu.com/v1/c4b02d61-tick-orange.svg" width="16" height="16" alt="Yes" />|<img src="https://assets.ubuntu.com/v1/c4b02d61-tick-orange.svg" width="16" height="16" alt="Yes" />|<img src="https://assets.ubuntu.com/v1/c4b02d61-tick-orange.svg" width="16" height="16" alt="Yes" />|
+|Subscriptions|Essential|Standard|Advanced|
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: The table is missing the "UA-I Level" cell

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 3]
```diff
@@ -9,413 +9,380 @@ context:
...
+3. Kernel Livepatch Service
+  1. The Kernel Livepatch Service includes a licence to use Canonical’s kernel livepatch daemon, access to available kernel livepatches, and support for the service.
+  2. The Kernel Livepatch Service provides security livepatches for selected High and Critical kernel CVEs. Some CVEs may be ineligible for livepatching due to technical limitations within the livepatching system or for other reasons.
+  3. The Kernel Livepatch Service may provide kernel non-security bug fixes as kernel livepatches.
+  4. Specific kernel versions are not guaranteed to receive livepatches indefinitely.  
+  5. Only the default LTS kernel is available for livepatching. This includes  its backport as the last HWE kernel to the previous LTS release.
+4. Extended Security Maintenance
+  1. Extended Security Maintenance provides available High and Critical CVE fixes for a number ofserver packages in the Ubuntu Main Repository. A complete list of packages included in Extended Security Maintenance for a given release can be found at: [https://wiki.ubuntu.com/SecurityTeam/ESM/](https://www.google.com/url?q=https://wiki.ubuntu.com/SecurityTeam/ESM/&sa=D&ust=1556713749211000)
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: s/ofserver/of server

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 4]
```diff
@@ -9,413 +9,380 @@ context:
...
+        3. Support must be purchased for all nodes in the supported Kubernetes cluster.
+        4. Supported versions of Kubernetes include the current stable minor release and the two most recent minor releases in the stable release channel. Additional information can be found at: [https://github.com/juju-solutions/bundle-canonical-kubernetes/wiki/Supported-Kubernetes-Versions](https://www.google.com/url?q=https://github.com/juju-solutions/bundle-canonical-kubernetes/wiki/Supported-Kubernetes-Versions&sa=D&ust=1556713749223000)
+        5. For any deployment of the Charmed Distribution of Kubernetes carried out by Canonical while under contract for a deployment, which result in the customisation of any Charms, the customisation will be supported for 90 days after completion of the deployment.
+    3. Unless Full-Stack Support requirements are met, support is limited to:
+        1. The software packages and Charms necessary for running the Charmed Distribution of Kubernetes.
+        2. The software packages available from apt.kubernetes.io and Kubernetes clusters deployed using kubeadm.
+        3. Bug-fix support in the supplied software artefacts by Canonical.
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: Copy doc has "artifacts"

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 5]
```diff
@@ -9,413 +9,380 @@ context:
...
+5. Service conclusion. At the end of the service term, the Managed Service will initiate an operational transfer. Operational transfer includes:
+  1. hand over of all credentials of the hosts and management software to the customer.
+  2. hand over of the administrative credentials of Landscape (The continued operation of Landscape is subject to purchase and agreement of appropriate license terms.)
+  3. coordination of any applicable training (if purchased).
+6. Customer dependencies. The Managed Service requires:
+  1. continuous VPN access for Canonical support personnel to the Environment;
+  2. utilisation parameters per Node to be kept below the maximum specified in the design document provided by Canonical when the Environment is delivered to the customer;
+  3. the facility where the Environment is hosted to comply with the minimum required measures to function, including but not limited to, connectivity, sufficient power supply, sufficient cooling system, and physical access control to the environment; and
+  4. that the Minimum Size Requirement for the Cloud or Cluster is maintained at all times.
+7. Uptime Service Level
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: This one is being wrapped in a `p` tag which is pushing it off the number

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 6]
```diff
@@ -9,413 +9,380 @@ context:
...
-    2. OpenStack support.
-    3. MAAS support.
-    4. Providing native images for a chosen hypervisor.
-    5. Additionalexclusions match those of the applicable Ubuntu Advantage for Infrastructure service offering.
+6. The Ubuntu Advantage for Infrastructure, Virtual service does not provide:
+  1. Hypervisor support.
+  2. OpenStack support.
+  3. MAAS support.
+  4. Providing native images for a chosen hypervisor.
+  5. Additionalexclusions match those of the applicable Ubuntu Advantage for Infrastructure service offering.
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: s/Additionalexclusions/Additional exclusions

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 7]
```diff
@@ -9,413 +9,380 @@ context:
...
-    3. Application containers.
-    4. Dual-booting (cohabitating with other operating systems).
-    5. Peripherals which are not certified to work with Ubuntu.
-    6. Support for non-desktop packages.
-    7. Community flavours of Ubuntu.
-    8. Support for architectures other than x86\_64.
-    9. Developer tools.
-    10. Support for packages not in the base Ubuntu desktop image.
+1. Overview. Ubuntu Advantage for Desktop services match those of the applicable Ubuntu Advantage for Infrastructure, subject to the exceptions listed below.
+2. Scope. Ubuntu Advantage for Desktop covers all packages in the base Ubuntu desktop image and packages necessary for basic network authentication and connectivity using sssd, winbind,network-manager, and network-manager plugins in the Ubuntu Main repository.
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: s/winbind,network-manager/winbind, network-manager

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 8]
```diff
@@ -9,413 +9,380 @@ context:
...
-        1. Support beyond that of the Livepatch on Premises packages.
-        2. Licenses to register machines to consume livepatches.
-    4. Each release of Livepatch on Premises will be supported for 1 year from its release date.
+1. Landscape on Premises
+  1. Landscape on Premises support is provided at the Advanced response times.
+  2. The Landscape on Premises service provides:
+    1. Licence to download and install a single instance of Landscape on Premises.
+    2. Ability to use Landscape on-premises management and monitoring services for machines (whether physical or virtual) for which the customer has purchased Ubuntu Advantage services.
+    3. Deployment methods:
+      1. When installed using the “Quickstart” install method, support is included for the machine on which Landscape on Premises is installed.
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: This one should be `1.2.3.1`

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 9]
```diff
@@ -9,413 +9,380 @@ context:
...
-        6. Participate in regular review calls addressing the customer's operational issues.
-        7. Organise multi-vendor issue coordination through TSANet or Canonical's direct partnerships where applicable. When the root cause is identified, the DSE will work with the vendor for that sub-system, working to resolve the case through their normal support process.
-        8. Attend applicable Canonical internal training and development activities (in-person and remote).
-    2. Canonical will hold a quarterly service review meeting with the customer to assess service performance and determine areas of improvement.
-    3. The DSE is available to respond to support cases during the DSE’s working hours. Outside of business hours, support will be provided per the Ubuntu Advantage Support Process.
-    4. If a DSE is on leave for longer than five consecutive business days, Canonical will assign a temporary remote resource to cover the leave period. Canonical will coordinate with the customer with respect to foreseeable DSE leave.
+<h2 id="uasd-professional-support">Professional Support Services</h2>
+
+1. Technical Account Manager (TAM)
+  1. Canonical will enhance
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: Not in the copy doc

Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: This one is reset back to 1.1 when it should be 1.3

Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/index.md]: support-manager@canonical.com could be a link

File: templates/legal/ubuntu-advantage-service-description/index.md (hunk 4 above)
Comment (pmahnke) [reply templates/legal/ubuntu-advantage-service-description/index.md]: english spelling

**Pull request merged**